src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexReadOperation.cs

```diff
@@ -469,7 +469,7 @@ public override SortedSet<string> Terms(string field, string fromValue, long pag
         }
 
         public override IEnumerable<BlittableJsonReaderObject> IndexEntries(IndexQueryServerSide query, Reference<int> totalResults,
-            DocumentsOperationContext documentsContext, Func<string, SpatialField> getSpatialField, bool ignoreLimit, CancellationToken token)
+            DocumentsOperationContext documentsContext, Func<string, SpatialField> getSpatialField, bool ignoreLimit, CancellationToken token) //TODO stav: ignoreLimit unused
```

Comment:
@arekpalinski @maciejaszyk can you comment on this?

Reply:
Well, for Corax this is true. In Lucene, you've to retrieve all data to get raw entries. In Corax, we can easily do pagination.

Reply:
User can send an `ignoreLimit` param and I forward it to all shards. Not sure what the meaning will be if corax doesn't use this